docs/src/modules/components/TopLayoutBlog.js

```diff
@@ -230,6 +230,8 @@ function TopLayoutBlog(props) {
         disableAlternateLocale
         card={card}
         type="article"
+        authors={headers.authors ? headers.authors.map((key) => authors[key].name) : undefined}
+        publicationDate={headers.date ? headers.date.slice(0, 'xxxx-xx-xx'.length) : undefined}
```

Comment:
The published date on https://www.linkedin.com/post-inspector/inspect/https:%2F%2Fretool.com%2Fblog%2Fgit-branching-with-source-control%2F seem to work, maybe we need the whole date
```suggestion
        publicationDate={headers.date ? headers.date : undefined}
```